package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lyra-network/openapi-dev-tool",
-  "version": "9.7.0",
+  "version": "9.8.0",
```

Comment:
Please use semver semantic. Indeed, if the goal of your PR is to fix an issue, you have to use 9.7.1